TransformationSystem/Agent/TransformationCleaningAgent.py

```diff
@@ -142,21 +142,11 @@ def execute( self ):
         if transDict[ 'Type' ] in self.dataManipTTypes:
           res = self.archiveTransformation( transDict['TransformationID'] )
           if not res['OK']:
-<<<<<<< Updated upstream
```

Comment:
We could try to spot this with post commit/post merge hooks, or even in the CI tests